chia/full_node/mempool.py

```diff
@@ -403,15 +404,30 @@ def create_bundle_from_mempool_items(
                 continue
             try:
                 cost = uint64(0 if item.npc_result.conds is None else item.npc_result.conds.cost)
-                unique_coin_spends, cost_saving, unique_additions = eligible_coin_spends.get_deduplication_info(
-                    bundle_coin_spends=item.bundle_coin_spends, max_cost=cost
-                )
+                if skipped_items >= 3:
+                    # Process this item straight away (no eligbility consideration)
+                    log.info("Processing mempool item without eligbility consideration: %s", name.hex())
+                    unique_coin_spends = []
+                    unique_additions = []
+                    for spend_data in item.bundle_coin_spends.values():
+                        unique_coin_spends.append(spend_data.coin_spend)
+                        unique_additions.extend(spend_data.additions)
+                    cost_saving = 0
+                else:
+                    unique_coin_spends, cost_saving, unique_additions = eligible_coin_spends.get_deduplication_info(
+                        bundle_coin_spends=item.bundle_coin_spends, max_cost=cost
+                    )
                 item_cost = cost - cost_saving
                 log.info("Cumulative cost: %d, fee per cost: %0.4f", cost_sum, fee / item_cost)
                 if (
                     item_cost + cost_sum > self.mempool_info.max_block_clvm_cost
                     or fee + fee_sum > DEFAULT_CONSTANTS.MAX_COIN_AMOUNT
                 ):
+                    # Let's skip this one
+                    skipped_items += 1
+                    if skipped_items < 20:
```

Comment:
I think we should also break out of the loop here if `item_cost` is less than some threshold. That would mean that we're getting very close to the limit anyway, and maybe we won't realistically find a transaction small enough.

i.e.

```
if skipped_items < 20 and item_cost > 6000000:
```

the 6 million cut-of comes from: https://www.chia.net/2024/01/11/getting-to-know-the-mempool-and-transaction-fees/